lib/compiler-cranelift/src/translator/code_translator.rs

```diff
@@ -1535,6 +1646,21 @@ pub fn translate_operator<FE: FuncEnvironment + ?Sized>(
             let (a, b) = pop2_with_bitcast(state, type_of(op), builder);
             state.push1(builder.ins().fmin(a, b))
         }
+        Operator::F32x4PMax | Operator::F64x2PMax => {
+            unimplemented!("SIMD Operator {:?} not yet implemented", op);
+            // let (a, b) = pop2_with_bitcast(state, type_of(op), builder);
```

Comment:
commented-out code here and a few lines later

Reply:
It will be useful from the future (is directly ported from cranelift-wasm), that's why is useful keeping it!